scripts/maintenance/migrate_project/src/db.py

```diff
@@ -0,0 +1,163 @@
+from collections import deque
+from contextlib import contextmanager
+from typing import Any, Deque, Dict, Iterator, Optional, Tuple
+from uuid import UUID
+
+from models import DBConfig
+from simcore_postgres_database.models.file_meta_data import file_meta_data
+from simcore_postgres_database.models.projects import projects
+from sqlalchemy import and_, create_engine, select
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.engine.base import Connection
+from sqlalchemy.engine.result import ResultProxy
+
+
+@contextmanager
+def db_connection(db_config: DBConfig) -> Iterator[Connection]:
+    engine = create_engine(
+        f"postgresql://{db_config.user}:{db_config.password}@{db_config.address}/{db_config.database}",
+        echo=True,
+    )
+    with engine.connect() as con:
+        yield con
+
+
+def _project_uuid_exists_in_destination(
+    connection: Connection, project_id: str
+) -> bool:
+    statemet = select([projects.c.id]).where(projects.c.uuid == f"{project_id}")
+    exits = len(list(connection.execute(statemet))) > 0
+    return exits
+
+
+def _meta_data_exists_in_destination(connection: Connection, file_uuid: str) -> bool:
+    statemet = select([file_meta_data.c.file_uuid]).where(
+        file_meta_data.c.file_uuid == f"{file_uuid}"
+    )
+    exits = len(list(connection.execute(statemet))) > 0
+    return exits
+
+
+def _get_project(connection: Connection, project_uuid: UUID) -> ResultProxy:
+    statemet = select([projects]).where(projects.c.uuid == f"{project_uuid}")
+    return connection.execute(statemet)
+
+
+def _get_hidden_project(connection: Connection, prj_owner: int) -> ResultProxy:
+    statemet = select([projects]).where(
+        and_(projects.c.prj_owner == prj_owner, projects.c.hidden == True)
+    )
+    return connection.execute(statemet)
```

Comment:
actually it's probably more readable to insert the query directly in the execute call